<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.FieldExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-05-02T08:34:11" hostname="testing-docker-f5468bb5-c282-4c60-9f7e-ab0b1a158756" time="0.006">
  <properties/>
  <testcase name="Exception should have proper formatted message[0]" classname="griffon.exceptions.FieldExceptionSpec" time="0.004"/>
  <testcase name="Exception should have proper formatted message[1]" classname="griffon.exceptions.FieldExceptionSpec" time="0.0"/>
  <testcase name="Exception should have proper formatted message[2]" classname="griffon.exceptions.FieldExceptionSpec" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
